client-src/elements/chromedash-report-review-latency-page.ts

```diff
@@ -86,13 +86,15 @@ export class ChromedashReportReviewLatencyPage extends LitElement {
   }
 
   renderFeatureReviewsRow(reviewLatency) {
+    const groupedByGateType = Object.groupBy(reviewLatency.gate_reviews, ({gate_type}) => gate_type);
     const gateLatency = {};
-    for (const gr of reviewLatency.gate_reviews) {
+    for (const gateType in groupedByGateType) {
+      const gr = groupedByGateType[gateType][0]; // Assuming one review per gate_type for a feature
```

Comment:
I wonder where this assumption comes from, and I don't think that it is true.  At least it added a comment to mention it.